dpos/manager/dposmanager.go

```diff
@@ -267,6 +278,7 @@ func (d *DPOSManager) OnPong(id dpeer.PID, height uint32) {
 
 func (d *DPOSManager) OnBlock(id dpeer.PID, block *types.Block) {
 	log.Info("[ProcessBlock] received block:", block.Hash().String())
+	delete(d.requestedBlocks, block.Hash())
```

Comment:
Show warn log if received block not in requested list.

Reply:
fixed